src/Robo/Plugin/Commands/DevelopmentModeCommands.php

```diff
@@ -142,14 +142,15 @@ public function databaseRefreshDdev(string $siteName = 'default', array $options
             $dbPath = $this->databaseDownload($siteName);
         }
 
-        $this->io()->section("importing $siteName database.");
-        $this->say("Importing $dbPath");
-        $this->taskExec(LocalDevEnvironmentTypes::DDEV->value)
-            ->arg('import-db')
-            ->option('database', $siteName === 'default' ? 'db' : $siteName)
-            ->option('file', $dbPath)
+        $this->io()->section("refreshing $siteName database.");
+        $this->say("Dropping existing database for $siteName");
+        $this->taskExec('drush')
+            ->arg('sql:drop')
+            ->option('uri', $siteName)
+            ->option('yes')
             ->run();
-
+        $this->say("Importing $dbPath");
+        $this->_exec("zcat '$dbPath' | drush sql:cli --uri=$siteName");
```

Comment:
Here's the big drawback with loading the db inside the container: We are effectively transferring the data over a virtual network interface from one container to another, which is probably noticeably slower than loading the data into the database from within the database container itself, which is what the import-db command does. 

Now, we could try to run this in the database container, but it would require mounting the benz-platform directory into the database container, which ddev does not do by default. 

To do this "right" we would have to:
1. Tell ddev to mount our application root directory into the database server somewhere, so that the database server has our db files accessible to it. 
2. Write a custom db command (not hard*) to load the data into the database. 

* There is _one_ part that is hard, and that is finding the correct authentication information. Granted, we can hard-code it in the command since it's always the same in ddev for all the databases, but the right way would be to get the connection info either by parsing the php settings, or (better) from `drush sql:connect` but if you run drush sql:connect, you need a fully working php/drupal environment installed into the database container. So that's a no-go. 

For these reasons, I went with just using drush, as above, even though it is transferring the data between containers.